src/Control/Monad/Linear/Internal.hs

```diff
@@ -30,37 +31,59 @@ import qualified Control.Monad.Trans.State.Strict as NonLinear
 
 -- TODO: make the laws explicit
 
--- | Enriched linear functors.
+-- | Enriched linear functors. These are control functors. The functor of type
```

Comment:
Remove any reference to enriched functors (prefer control functors) from the documentation. It's just an artifact of the fact that I didn't have names for them, then, and I used the mathematical terminology.

I'll make a short reference to enriched functors in a mathematical note in the future. For reference, but it is not relevant for most of the documentation, as I don't want to use this terminology anymore.